src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -276,6 +276,21 @@ namespace JSONHelpers
         }
     }
 
+    std::vector<GUID> FancyZonesData::GetVirtualDeslktopIds() const
+    {
+        std::vector<GUID> ids;
+        for (auto it = deviceInfoMap.begin(); it != deviceInfoMap.end(); ++it)
```

Comment:
Maybe to keep code cleaner we could just iterate like `for (const auto& [id, info] : deviceInfoMap)`